test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,11 +36,35 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void elementsHaveNull() throws Exception {
```

Comment:
It is better to follow the convention that the names of the method being tested and the test method are the same, at least for the sake of better readability and tracking.